dev/devicelab/bin/tasks/plugin_lint_mac.dart

```diff
@@ -473,7 +473,7 @@ void _validateMacOSPodfile(String appPath) {
   if (!podfileLockOutput.contains(':path: Flutter/ephemeral\n')
       || !podfileLockOutput.contains(':path: Flutter/ephemeral/.symlinks/plugins/url_launcher_macos/macos')
       || !podfileLockOutput.contains(':path: Flutter/ephemeral/.symlinks/plugins/test_plugin_swift/macos')
-      || podfileLockOutput.contains('url_launcher/')) {
+      || !podfileLockOutput.contains('url_launcher/')) {
```

Comment:
The spirit of this check was to make sure the `url_launcher` framework was not embedded in the app.  It was using the Podfile.lock as a shortcut for this.  A more appropriate test now would be to validate that the generated app contains `Contents/Frameworks/url_launcher_macos.framework` but not `Contents/Frameworks/url_launcher.framework`.

Reply:
got it. I added another check to the test for the time being. Do you have any suggestion about how to verify the frameworks?

Reply:
Something like
```dart
  final String appDirectory = path.join(
    appPath,
    'build',
    'macos',
    'Build',
    'Products',
    'Release',
    '${path.basename(appPath)}.app',
  );

  final String frameworksDirectory = path.join(
    appDirectory,
    'Contents',
    'Frameworks',
  );

  checkDirectoryExists(path.join(
    frameworksDirectory,
    'url_launcher_macos.framework',
  ));

  checkDirectoryNotExists(path.join(
    frameworksDirectory,
    'url_launcher.framework',
  ));
```

There also may be a more appropriate test location for that check, like https://github.com/flutter/flutter/blob/master/packages/flutter_tools/test/integration.shard/macos_content_validation_test.dart